main.go

```diff
@@ -81,32 +89,54 @@ func printVersion() {
 	fmt.Println(version)
 }
 
-func generateFromRequest(pkgDir, structModelName, repositoryInterfaceName string) (string, error) {
-	fset := token.NewFileSet()
-	dir, err := parser.ParseDir(fset, pkgDir, nil, parser.ParseComments)
-	if err != nil {
-		return "", err
-	}
-
-	pkg, err := code.ParsePackage(dir)
-	if err != nil {
-		return "", err
-	}
-
-	return generateRepository(pkg, structModelName, repositoryInterfaceName)
-}
-
 var (
 	errStructNotFound    = errors.New("struct not found")
 	errInterfaceNotFound = errors.New("interface not found")
 )
 
-func generateRepository(pkg code.Package, structModelName, repositoryInterfaceName string) (string, error) {
-	structModel, ok := pkg.Structs[structModelName]
-	if !ok {
-		return "", errStructNotFound
+func generateFromRequest(
+	pkgDir, modelDir, structModelName, repositoryInterfaceName, destPkgName string,
+) (string, error) {
```

Comment:
This starts to be a bit complicated. Can you wrap it inside a struct? Make `generateRepository` a method inside the struct too because it also needs access to certain variables.

Reply:
yeah..I understand your concern. lemme think about it.

Reply:
I got the idea like this 2c0444d5efee8d95e2a3b4f7e3ebd41b97c0b8e7

Reply:
That looks okay. But I am thinking about replacement for the word "Flow". If someone reads the code, he will not know exactly what this struct does.

Reply:
yeah I agree "Flow" is a bad name 😅. I think it sounds like a "generator" (in larger context) but that term is already used.